src/types.ts

```diff
@@ -0,0 +1,20 @@
+// external
+import * as t from 'io-ts';
+
+// main
+import { FixedLengthArray } from '@transcend-io/type-utils';
+
+// local
+import type { TrackingPurpose } from './core';
+
+/** Regime purpose scopes configuration */
+export const RegimePurposeScopesConfig = t.array(
+  FixedLengthArray(2, 2, t.array(t.string)),
+);
+/** Type override */
```

Comment:
@linh-transcend nit but this can go into `core` ?

Reply:
Just moved! it belongs there anyway after a second thought